### DISCUSSION TITLE
Regarding Apollo Build

### DISCUSSION THREAD

**Discussion by Saddy21** [guardstrikelab/carla_apollo_bridge#37] [Q&A] [answered]
I am following the documentation for CARLA APOLLO BRIDGE
Until 
 _**sudo rm -rf /apollo/.cache
bash docker/scripts/dev_start.sh**_ everything is going fine
But after running: 
**_./apollo.sh build_gpu_**
It shows **pip._vendor.urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='pypi.tuna.tsinghua.edu.cn', port=443): Read timed out.** 
 every time.
Also i noticed it is using python 3.6 version,is it because of that?
Is there any way to change it to python 3.8?
Please help
![Screenshot from 2023-01-04 18-41-48](https://user-images.githubusercontent.com/98169680/210562269-a9a2c5ad-b025-421e-8f4a-2faa8ff24fe2.png)

**Comment by wohu1104** [comment, marked as answer]
You can replace the pip source as the default source, do not use pypi.tuna.tsinghua.edu.cn this source, or try other source.

**Reply by Saddy21** [reply]
Thank you very much.
